service_container/parent_services.rst

```diff
@@ -147,7 +147,7 @@ All attributes on the parent service are shared with the child **except** for
 
 .. note::
 
-    If you have a ``_defaults`` section in your file, all child services are required
+    If you have a ``_defaults`` section in your ``services.yaml``, all child services are required
```

Comment:
"[...] in your ``services.yaml`` file [...]"?

Reply:
:-)